include/lbann/data_readers/compound_data_reader.hpp

```diff
@@ -92,12 +92,14 @@ class generic_compound_data_reader : public generic_data_reader {
     }
   }
 
+/*
   void swap_role(std::string role) override {
     generic_data_reader::swap_role(role);
     for (auto&& reader : m_data_readers) {
       reader->swap_role(role);
     }
   }
+*/
```

Comment:
Can you just delete this instead of commenting it out?

Reply:
Will do